jina/peapods/zmq/__init__.py

```diff
@@ -107,11 +107,11 @@ def _register_pollin(self):
             self.poller.register(self.in_connect_sock, zmq.POLLIN)
 
     def pause_pollin(self):
-        """Remove :attr:`in_sock` from the poller """
+        """Remove :attr:`in_sock` from the poller"""
```

Comment:
Lets remove the changes in this file from the PR? I think they are only linting and this PR should not touch this file it appears

Reply:
reverted using [git restore](https://stackoverflow.com/a/59756673) :)